website2/src/app/clean-air-forum/logistics/page.tsx

```
@@ -5,56 +5,90 @@ import React from 'react';
+        <>
+          <Divider className="bg-black p-0 m-0 h-[1px] w-full" />
+          <div>
+            <div className="flex flex-col md:flex-row md:space-x-8">
+              <div className="md:w-1/3 mb-4 md:mb-0">
+                <h2 className="text-2xl font-bold text-gray-900">
+                  Vaccination
+                </h2>
+              </div>
+              <div
+                className="md:w-2/3 space-y-4"
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)